.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "*-v*.*.*"
+
+# We need this to be able to create releases.
+permissions:
+  contents: write
+
+jobs:
+  create-release:
+    name: create-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+
+      - name: Show the tag
+        run: |
+          echo "tag is: ${{ github.ref_name }}"
```

Comment:
Why would the tag be ref_name? It's automatically set on push: tags ?

Reply:
yes, afaik